cluster/gce/configure-vm.sh

```diff
@@ -74,8 +74,37 @@ function set-broken-motd() {
   echo -e '\nBroken (or in progress) Kubernetes node setup! Suggested first step:\n  tail /var/log/startupscript.log\n' > /etc/motd
 }
 
-function set-good-motd() {
-  echo -e '\n=== Kubernetes node setup complete ===\n' > /etc/motd
+function reset-motd() {
+  # kubelet is installed both on the master and nodes, and the version is easy to parse (unlike kubectl)
+  local -r version="$(/usr/local/bin/kubelet --version | cut -f2 -d\ )"
```

Comment:
Older versions of the kubelet want `--version=true` here. Do we care if this only works for 1.2+?